app/Http/Controllers/MediumController.php

```diff
@@ -40,38 +41,61 @@ public function store(Request $request)
     /**
      * Display the specified resource.
      *
-     * @param  int  $id
+     * @param int $id
      * @return \Illuminate\Http\Response
      */
+
+    /**
+     * @OA\Get(
+     *      path="/media/{id}",
+     *      operationId="getMediumById",
+     *      tags={"Media"},
+     *      summary="Get medium information",
+     *      description="Returns medium data",
+     *      @OA\Parameter(
+     *          name="id",
+     *          description="Medium id",
+     *          required=true,
+     *          in="path",
+     *          @OA\Schema(
+     *              type="integer"
+     *          )
+     *      ),
+     *      @OA\Response(
+     *          response=200,
+     *          description="Successful operation",
+     *       ),
+     *      @OA\Response(
+     *          response=400,
+     *          description="Bad Request"
+     *      ),
+     *      @OA\Response(
+     *          response=401,
+     *          description="Unauthenticated",
+     *      ),
+     *      @OA\Response(
+     *          response=403,
+     *          description="Forbidden"
+     *      )
+     * )
+     */
     public function show($id)
     {
         $medium = $this->mediumService->findById($id);
-        if($medium instanceof Medium){
+        if ($medium instanceof Medium) {
             return $this->sendResponse(new MediumResource($medium), 'Medium retrieved successfully.');
-        }
-        else {
+        } else {
             return $this->sendError($medium);
         }
     }
 
-    /**
-     * Update the specified resource in storage.
-     *
-     * @param  \Illuminate\Http\Request  $request
-     * @param  int  $id
-     * @return \Illuminate\Http\Response
-     */
+
```

Comment:
Why are you removing the default comments below the show method and not removing above. Should they be removed or left all?
One convention but everything stays or nothing stays.